modules/pam_faillock/pam_faillock.c

```diff
@@ -620,7 +620,17 @@ faillock_message(pam_handle_t *pamh, struct options *opts)
 		if (left > 0) {
 			left = (left + 59)/60; /* minutes */
 
-			pam_info(pamh, _("(%d minutes left to unlock)"), (int)left);
+#if defined HAVE_DNGETTEXT && defined ENABLE_NLS
+		pam_info(pamh, _(dngettext(PACKAGE,
+		"(%d minute left to unlock)",
+		"(%d minutes left to unlock)",
+		(int)left)));
```

Comment:
This line needs to look like this: `(int)left), (int)left);`

Reply:
Thanks, already figured that out due to the CI :)